aopy/data/bmi3d.py

```diff
@@ -864,21 +864,31 @@ def get_kinematics(preproc_dir, subject, te_id, date, samplerate, preproc=None,
             kinematics, samplerate = preproc(raw_kinematics, samplerate)
         else:
             kinematics = raw_kinematics
+
     else:
         kinematics, samplerate = get_task_data(preproc_dir, subject, te_id, date, datatype, 
                                                samplerate, preproc=preproc, **kwargs)
 
     return kinematics, samplerate
 
 def _get_kinematic_segment(preproc_dir, subject, te_id, date, start_time, end_time, samplerate, 
-                          preproc=None, datatype='cursor', **kwargs):
+                          preproc=None, datatype='cursor', return_nan=False, **kwargs):
```

Comment:
can you also add and document this argument in `get_kinematic_segments` and `tabulate_kinematic_data`

Reply:
both of those functions call _get_kinematic_segment so return_nan should be able to be passed as kwarg. The test uses tabulate_kinematic_data and shows this.